service/controller/app/v1/resource/chart/resource.go

```diff
@@ -75,16 +75,27 @@ func (r *Resource) Name() string {
 }
 
 // equals asseses the equality of ReleaseStates with regards to distinguishing fields.
-func equals(a, b *v1alpha1.Chart) bool {
-	if a.Name != b.Name {
+func equals(current, desired *v1alpha1.Chart) bool {
```

Comment:
In the `desired` chart, we may have annotations value we want to deploy. 
So for that, I would like to call these parameters explicitly rather than a, b.